fluentlenium-core/src/main/java/org/fluentlenium/core/action/FillConstructor.java

```diff
@@ -47,4 +47,9 @@ public FillConstructor with(String... values) {
         }
         return this;
     }
+
+    public FillConstructor withValues(String... values) {
```

Comment:
My mistake, with doesn't have unit test as well. So I can merge this change as well